share/spice/economic_indicators/economic_indicators.js

```diff
@@ -0,0 +1,97 @@
+(function (env) {
+    "use strict";
+    function getDataFromAPIResponse(api_response){
+        //validate if api response is null
+        if(api_response === null || api_response === undefined){
+            return null;
+        }
+        //the root object of api response must be an array of length 2
+        if (Object.prototype.toString.call( api_response ) !== '[object Array]' || api_response.length != 2){
```

Comment:
This is done as suggested https://github.com/gauravtiwari5050/zeroclickinfo-spice/commit/8270b5957238fe90716bc98025efe09d9157b4e6